doc/user-manual/tools/emacs-mode.rst

```diff
@@ -9,7 +9,8 @@ Agda programs are commonly edited using `Emacs
 section.  Other editors with interactive support for Agda include Atom
 (`agda-mode on Atom <https://atom.io/packages/agda-mode>`_), Visual
 Studio Code (`agda-mode on VS Code
-<https://github.com/banacorn/agda-mode-vscode>`_), and Vim (`agda-vim
+<https://github.com/banacorn/agda-mode-vscode>`_), Neovim (`Cornelis
+<https://github.com/isovector/cornelis>`_), and Vim (`agda-vim
```

Comment:
Maybe this enumeration would be easier to read as a bullet list.

If you are willing to change this to a bullet list, that would be great, but if not, also fine. (Don't want to feature creep this PR.)

Reply:
I'll change it :)